logging/src/main/java/org/jboss/as/logging/LoggingOperations.java

```diff
@@ -79,6 +86,13 @@ public static PathAddress getAddress(final ModelNode operation) {
      */
     static void addCommitStep(final OperationContext context, final ConfigurationPersistence configurationPersistence) {
         context.addStep(new CommitOperationStepHandler(configurationPersistence), Stage.RUNTIME);
+        final Collection<OperationStepHandler> afterCommitSteps = context.getAttachment(AFTER_COMMIT_STEPS);
+        if (afterCommitSteps != null && !afterCommitSteps.isEmpty()) {
+            for (OperationStepHandler step : afterCommitSteps) {
+                context.addStep(step, Stage.RUNTIME);
+            }
+            context.addStep(new CommitOperationStepHandler(configurationPersistence), Stage.RUNTIME);
```

Comment:
No it should be inside the if block. I only need to add another commit step if there are other changes to commit.